src/ds5/ds5-device.cpp

```diff
@@ -649,8 +649,19 @@ namespace librealsense
 
     std::vector<uint8_t> ds5_device::get_raw_calibration_table(ds::calibration_table_id table_id) const
     {
-        command cmd(ds::GETINTCAL, table_id);
-        return _hw_monitor->send(cmd);
+        std::vector<byte> res;
```

Comment:
consider override this function in the new ds5-safety class instead if if..else..
Or maybe add a parameter to this function